DESCRIPTION

```diff
@@ -20,12 +20,29 @@ RoxygenNote: 7.0.2
 Packaged: 2018-09-19 21:17:11 UTC; jared
 Repository: CRAN
 Date/Publication: 2018-09-19 22:20:02 UTC
-Imports: Rcpp, RcppArmadillo, RcppParallel, coda, Hmisc, parallel, doParallel, foreach, matrixStats
+Imports: 
+    Rcpp,
+    RcppArmadillo,
+    RcppParallel,
+    coda (>= 0.19.3),
+    Hmisc,
+    parallel,
+    doParallel,
+    foreach,
+    matrixStats,
+    methods
 Suggests: 
     testthat,
     spelling,
     knitr,
-    rmarkdown
+    rmarkdown,
+    latex2exp,
+    ggplot2,
+    rpart,
+    rpart.plot
 SystemRequirements: GNU make
 Language: en-US
 VignetteBuilder: knitr
+Remotes: 
+    mathematica-mpr/coda
+Encoding: UTF-8
```

Comment:
@ignacio82 Was the "encoding" line what ended up fixing your problem? I see an "undefined" symbol error.

I'm wondering what fixed you're bug vs what was general fixes for devtools:check()

Reply:
commit message with the relevant link:
https://github.com/ignacio82/bcf/commit/ddf96255963c8986b487fba57103f64d4ecead45